### PR TITLE
Rfairooz/general meetings

### LINKED ISSUE #166: Add General Meetings to Events
- Ask David Ryan (Event Chair) for the times/place/title of the general meetings and put them into the list of ongoing events
- Ensure to follow the previous events in terms of their format and information

### PR DESCRIPTION
**Pull request by rfairooz**
Screen Shot 2022-09-06 at 4 10 43 PM

Screen Shot 2022-09-06 at 8 16 37 PM

### CODE REVIEW

File: components/Events.js [hunk 1]
```diff
@@ -6,6 +6,15 @@ const filters = {
 };
 
 export const Events = [
+	{
+		title: "Intern/UG Research Panel",
```
Comment (shahdivyank) [inline components/Events.js]: Intern/Research Panel

File: components/Events.js [hunk 2]
```diff
@@ -14,6 +23,15 @@ export const Events = [
 		end: new Date(2022, 9, 7, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Metaverse With Meta: Meta Panel",
```
Comment (shahdivyank) [inline components/Events.js]: Metaverse

File: components/Events.js [hunk 3]
```diff
@@ -62,6 +89,15 @@ export const Events = [
 		end: new Date(2022, 10, 18, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Running a Coding YouTube Channel: Interview with Namanh Kapur",
```
Comment (shahdivyank) [inline components/Events.js]: Interview with Namanh Kapur

File: components/Events.js [hunk 4]
```diff
@@ -6,6 +6,15 @@ const filters = {
...
 
 export const Events = [
+	{
+		title: "Intern/UG Research Panel",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 8, 27, 16, 0, 0),
+		end: new Date(2022, 8, 27, 18, 0, 0),
+		description:
+			"Are you someone searching for internships or research opportunities? For our first GM, we’ll be holding our annual intern & research panel. This is a great opportunity to ask questions and hear advice from students that have landed internship and undergraduate research positions.",
```
Comment (shahdivyank) [inline components/Events.js]: Learn about the tips and tricks undergraduates use to secure internships and research opportunities.

File: components/Events.js [hunk 5]
```diff
@@ -14,6 +23,15 @@ export const Events = [
...
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Metaverse With Meta: Meta Panel",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 9, 11, 16, 0, 0),
+		end: new Date(2022, 9, 11, 18, 0, 0),
+		description:
+			"The metaverse is a concept about which we know extremely little. Is it the future of the internet? Or a video game? Or a deeply uncomfortable version of Zoom? Join us for our second GM, as we’ll be holding a panel with Meta software engineers, discussing what is the metaverse, how social interaction will change, and their experience working under a MANGA company.",
```
Comment (shahdivyank) [inline components/Events.js]: Explore the future of the metaverse with Meta software engineers and learn about its impacts in social interaction.

File: components/Events.js [hunk 6]
```diff
@@ -46,6 +64,15 @@ export const Events = [
...
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Thinking Like A Programmer",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 10, 8, 16, 0, 0),
+		end: new Date(2022, 10, 8, 18, 0, 0),
+		description:
+			"When getting in computer science, everyone learns a new programming language or how to develop their first web app. But despite all the leetcoding and CS finals, no one wonders how to THINK like a programmer. If you’re interested in learning how to think like one, come to our third GM where we’ll discuss with a professor(TBA) on tips and tricks for thinking like a programmer.",
```
Comment (shahdivyank) [inline components/Events.js]: Discover the tips and tricks to THINK like a programmer and help you excel in your classes, projects, and various opportunities.

File: components/Events.js [hunk 7]
```diff
@@ -62,6 +89,15 @@ export const Events = [
...
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Running a Coding YouTube Channel: Interview with Namanh Kapur",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 10, 22, 16, 0, 0),
+		end: new Date(2022, 10, 22, 18, 0, 0),
+		description:
+			"Youtube has over a million coding videos and counting, with thousands of CS professionals posting content about learning how to code or how to surpass and interview for a MANGA company. Join us for our final virtual GM as we hold an interview with Youtube content creator and Ex-Microsoft engineer Namanh Kapur, discussing his journey towards making videos on Youtube, his advice towards future software engineers entering the industry, and his cool tips on starting a Youtube channel and making content.",
```
Comment (shahdivyank) [inline components/Events.js]: Join Namanh Kapur, a content creator, discuss his journey and various challenges he faced along the way.

**Review by shahdivyank — APPROVED** [review]

**Review by pondohoo — APPROVED** [review]

**Pull request merged**